src/components/Actions.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import { setQuerySort } from '../redux/actions';
+
+interface Props {
+  setQuery: (query: string) => void;
+}
+
+const ActionsTemplate: FC<Props> = ({
+  setQuery,
+}) => (
+  <>
+    <button
+      className="button"
+      type="button"
+      onClick={() => setQuery('title')}
+    >
+      Sort by title
+    </button>
+
+    <button
+      className="button"
+      type="button"
+      onClick={() => setQuery('user')}
+    >
+      Sort by user
+    </button>
+
+    <button
+      className="button"
+      type="button"
+      onClick={() => setQuery('completed')}
+    >
+      Sort by completed
+    </button>
+
+    <button
+      className="button"
+      type="button"
+      onClick={() => setQuery('reverse')}
+    >
+      Reverse
+    </button>
+  </>
+);
+
+const mapStateToProps = (state: { queryReducer: QuerySort }) => {
+  return {
+    query: state.queryReducer.query,
+  };
+};
+
+const mapDispatchToProps = (dispatch: Dispatch) => {
+  return {
+    setQuery: (query: string) => dispatch(setQuerySort(query)),
+  };
+};
```

Comment:
make it easy
<img width="946" alt="Screenshot 2020-03-16 at 07 30 06" src="https://user-images.githubusercontent.com/36257947/76725904-53447600-6758-11ea-8fe6-c5bbdd618818.png">